.circleci/verbatim-sources/job-specs/pytorch-job-specs.yml

```
@@ -156,9 +156,10 @@ jobs:
             fi
             return 1
           }
+          DISABLE_PEER_VERIFY_NODESOURCE='-o -o \'Acquire::https::deb.nodesource.com::Verify-Peer "false";\''
```

I guess `s/-o -o/-o`, but even after than one bash does not like this string :(
Replacing single quotes with double help, but then apt-get does not recognize the command
```suggestion
          DISABLE_PEER_VERIFY_NODESOURCE="-o 'Acquire::https::deb.nodesource.com::Verify-Peer \"false\";'"
```